### PR TITLE
Define latency buckets boundaries in the API to allow UIs to use them when display data.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: core/src/main/java/com/google/instrumentation/trace/TraceExporter.java [hunk 1]
```diff
@@ -172,6 +171,99 @@ static TraceExporter getNoopTraceExporter() {
      */
     public abstract void unregisterSpanNamesForCollection(Collection<String> spanNames);
 
+    /**
+     * The latency buckets boundaries. Samples based on latency for successful spans (the status of
+     * the span has a canonical code different than {@link CanonicalCode#OK}) are collected in one
+     * of these latency buckets.
+     */
+    public enum LatencyBucketsBoundaries {
```
Comment (adriancole) [inline core/src/main/java/com/google/instrumentation/trace/TraceExporter.java]: by sharing with UI, do you mean sharing by convention or literally writing this to json or such?

File: core/src/main/java/com/google/instrumentation/trace/TraceExporter.java [hunk 2]
```diff
@@ -172,6 +171,99 @@ static TraceExporter getNoopTraceExporter() {
      */
     public abstract void unregisterSpanNamesForCollection(Collection<String> spanNames);
 
+    /**
+     * The latency buckets boundaries. Samples based on latency for successful spans (the status of
+     * the span has a canonical code different than {@link CanonicalCode#OK}) are collected in one
```
Comment (bogdandrutu) [inline core/src/main/java/com/google/instrumentation/trace/TraceExporter.java]: Done.

File: core/src/main/java/com/google/instrumentation/trace/TraceExporter.java (hunk 1 above)
Comment (bogdandrutu) [inline core/src/main/java/com/google/instrumentation/trace/TraceExporter.java]: Correct.

Comment (bogdandrutu) [inline core/src/main/java/com/google/instrumentation/trace/TraceExporter.java]: Done.

File: core/src/main/java/com/google/instrumentation/trace/TraceExporter.java [hunk 3]
```diff
@@ -172,6 +171,99 @@ static TraceExporter getNoopTraceExporter() {
...
+       */
+      LatencyBucketsBoundaries(
+          long latencyLowerNs,
+          long latencyUpperNs,
+          String latencyLowerString,
+          String latencyUpperString) {
+        this.latencyLowerNs = latencyLowerNs;
+        this.latencyUpperNs = latencyUpperNs;
+        this.latencyLowerString = latencyLowerString;
+        this.latencyUpperString = latencyUpperString;
```
Comment (bogdandrutu) [inline core/src/main/java/com/google/instrumentation/trace/TraceExporter.java]: Done.

**Pull request merged**